tests/page-objects/contacts/admin-user.wdio.page.js

```diff
@@ -0,0 +1,68 @@
+const addUserButton = () => $('a#add-user');
+const addUserDialog = () => $('h2=Add user');
+const userName = () => $('#edit-username');
+const userFullName = () => $('#fullname');
+const userRole = () => $('#role');
+const userPlace = () => $('//span[@aria-labelledby="select2-facilitySelect-container"]');
+const userAssociatedContact = () => $('//span[@aria-labelledby="select2-contactSelect-container"]');
+const userPassword = () => $('#edit-password');
+const userConfirmPassword = () => $('#edit-password-confirm');
+const saveUserButton = () => $('a=Add user');
+const logoutButton = () => $('span=Log out');
+const select2SearchInputBox = () => $('//input[@aria-controls="select2-facilitySelect-results"]');
+const select2Name = () => $('.name');
+const select2SearchContactInputBox = () => $('//input[@aria-controls="select2-contactSelect-results"]');
+const select2AssociatedContact = () => $('.name');
```

Comment:
The `() => $('.name');` is used twice when defining elements. Let's reuse the existing one.

Reply:
Done